webapi/src/main/scala/org/knora/webapi/routing/v1/ResourcesRouteV1.scala

```diff
@@ -544,6 +590,68 @@ object ResourcesRouteV1 extends Authenticator {
                         loggingAdapter
                     )
             }
+
+        } ~  path("v1" / "resources" / "xml" ) {
+            post {
+                entity(as[NodeSeq]) {
+                    xml => requestContext =>
+                    val userProfile = getUserProfileV1(requestContext)
+
+                    val projectId = "http://data.knora.org/projects/DczxPs-sR6aZN91qV92ZmQ"
```

Comment:
Please change this to take the project IRI from the request, instead of using a hard-coded project.